kivakit-configuration/src/main/java/com/telenav/kivakit/configuration/lookup/RegistryKey.java

```diff
@@ -0,0 +1,11 @@
+package com.telenav.kivakit.configuration.lookup;
+
+import com.telenav.kivakit.kernel.language.values.identifier.StringIdentifier;
+
+class RegistryKey extends StringIdentifier
```

Comment:
What about use StringIdentifier directly as return type for InstanceIdentifier.key()?

Reply:
I think the class clarifies the use. It is package private, so it's not part of the API.